phpstan.neon

```diff
@@ -2,3 +2,10 @@ parameters:
     level: 5
     ignoreErrors:
         - '#Constant APP not found#'
+        - '#Constant DS not found#'
```

Comment:
For me it would be cleaner to load a bootstrap file that contains the constants.

Reply:
Applied.